beginner_source/nlp/advanced_tutorial.py

```diff
@@ -217,7 +217,7 @@ def _forward_alg(self, feats):
     def _get_lstm_features(self, sentence):
         self.hidden = self.init_hidden()
         embeds = self.word_embeds(sentence).view(len(sentence), 1, -1)
-        lstm_out, self.hidden = self.lstm(embeds)
+        lstm_out, self.hidden = self.lstm(embeds, self.hidden)
```

Comment:
Look at this line: https://github.com/pytorch/pytorch/blob/e694db0eeb3ff380b476e92710475efc4551c826/torch/nn/modules/rnn.py#L61